package.json

```diff
@@ -63,7 +63,7 @@
   },
   "dependencies": {
     "@amcharts/amcharts4": "^4.10.13",
-    "@anxolin/walletconnect-connector": "6.1.9",
```

Comment:
@josojo What would we have to have from this @anxolin library to adapt to this repository?

I have tested the minerva wallet with the new `web3-react` library, and found no problems.

Reply:
I think you don't need to use my library any more :) 
u are welcome to use it, but this was a temporal fix while they would accept my PR